app/src/main/java/fr/neamar/kiss/pojo/PojoComparator.java

```diff
@@ -3,8 +3,24 @@
 import java.util.Comparator;
 
 public class PojoComparator implements Comparator<Pojo> {
+
+    boolean bIncreasing;
+    public PojoComparator()
+    {
+        super();
+        bIncreasing = false;
+    }
+
+    public PojoComparator( boolean increasing )
```

Comment:
Just to be sure, this is always used as increasing right? Why create a custom property for it?
Or am I missing something?

Reply:
I was testing more matching algorithms and I needed reverse sorting for some of them. I suppose this can be removed.